commons-mongo/jvm/src/test/scala/com/avsystem/commons/mongo/SomethingPlain.scala

```diff
@@ -33,7 +33,10 @@ object SomethingPlain extends HasGenCodec[SomethingPlain] {
     string <- arbitrary[String]
     boolean <- arbitrary[Boolean]
     int <- arbitrary[Int]
-    long <- arbitrary[Long]
+    long <- Gen.oneOf(
+      Gen.choose(Long.MinValue, Int.MinValue - 1L),
+      Gen.choose(Int.MaxValue + 1L, Long.MaxValue),
+    )
```

Comment:
Why skip the "boring" integers?

Reply:
Because this test assumes that all of them will be `BsonInt64` which is no longer true (in general) after my changes.

Reply:
That won't be obvious in the future. At worst add a comment, at best update the test please.

Reply:
Update how? I would just remove `Long`s from this test altogether.